content/publishing-yaml/distribution.md

```diff
@@ -120,7 +120,7 @@ publishing:
 Publishing GitHub releases is available for GitHub repositories only.
```

Comment:
let's move GitHub releases out from the "Integrations" section and make it a level 2 heading:
## GitHub releases

Reply:
fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/340/commits/16cdc7401b6c4cc498c9b838674d2e7f42490cc2